### DISCUSSION TITLE
Cloudflare DNS is stating "could not find solver for dns-01"

### DISCUSSION THREAD

**Discussion by brantpeery** [0xJacky/nginx-ui#353] [Q&A]
How can I troubleshoot? 

here is the not so helpful log
Getting the certificate, please wait...

2024/04/11 04:34:12 [INFO] [Nginx UI] Generating private key for registering account

2024/04/11 04:34:12 [INFO] [Nginx UI] Preparing lego configurations

2024/04/11 04:34:12 [INFO] [Nginx UI] Creating client facilitates communication with the CA server

2024/04/11 04:34:12 [INFO] [Nginx UI] Setting DNS01 challenge provider

2024/04/11 04:34:12 [INFO] [Nginx UI] Setting environment variables

2024/04/11 04:34:12 [INFO] [Nginx UI] Registering user

2024/04/11 04:34:12 [INFO] acme: Registering account for brant.peery@gmail.com

2024/04/11 04:34:12 [INFO] [Nginx UI] Obtaining certificate

2024/04/11 04:34:12 [INFO] [domain] acme: Obtaining bundled SAN certificate

2024/04/11 04:34:12 [INFO] [cloud.atechinnovations.com] AuthURL: https://acme-v02.api.letsencrypt.org/acme/authz-v3/337076751667

2024/04/11 04:34:12 [INFO] [domain] acme: Could not find solver for: tls-alpn-01

2024/04/11 04:34:12 [INFO] [domain] acme: Could not find solver for: http-01

2024/04/11 04:34:12 [INFO] [domain] acme: Could not find solver for: dns-01

2024/04/11 04:34:12 [INFO] Deactivating auth: https://acme-v02.api.letsencrypt.org/acme/authz-v3/337076751667

**Comment by 0xJacky** [comment]
Please restart nginx-ui and try again.
